clients.go

```diff
@@ -63,6 +64,45 @@ func (u *Unifi) GetClientsDPI(sites []*Site) ([]*DPITable, error) {
 	return data, nil
 }
 
+// GetClientHistory pulls client history data from the controller.
+func (u *Unifi) GetClientHistory(sites []*Site, opts *ClientHistoryOpts) ([]ClientHistory, error) {
+	if opts == nil {
+		opts = NewClientHistoryOpts()
+	}
+	params := url.Values{}
+	params.Add("onlyNonBlocked", fmt.Sprint(opts.OnlyNonBlocked))
+	params.Add("includeUnifiDevices", fmt.Sprint(opts.IncludeUnifiDevices))
+	params.Add("withinHours", fmt.Sprint(opts.WithinHours))
```

Comment:
Recommend `strconv` instead of `fmt`. It's faster and trips fewer linters.

Reply:
Fixed with 4df8728